### PR TITLE
docs updates, including multi agent manager

### LINKED ISSUE #1627: Add detailed system graphic & p2p graphic
**Is your feature request related to a problem? Please describe.**
We don't have detailed graphical illustrations of the system and the p2p system in particular.

**Describe the solution you'd like**
Add

### PR DESCRIPTION
**Pull request by DavidMinarsch**
## Proposed changes

Docs updates

## Fixes

#1627

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

-

### CODE REVIEW

File: docs/multi-agent-manager.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## Adding agent instances
+
+``` python
+manager.add_agent("fetchai/weather_client:0.13.0")
+manager.add_agent("fetchai/weather_station:0.13.0")
+```
+
+missing:
+- configuring private keys
+- configure delayed startup sequence
```
Comment (DavidMinarsch) [inline docs/multi-agent-manager.md:35]: @marcofavorito and @solarw , the manager should support this so a demo like weather-skills.md is fully supported.

File: docs/acn.md [hunk 2]
```diff
@@ -4,4 +4,8 @@
   <p>Details coming soon. In the meantime check out <a href="../p2p-connection">this section.</a></p>
 </div>
 
+
+missing:
+- different configurations of setting up peer and client
```
Comment (DavidMinarsch) [inline docs/acn.md:9]: @DavidMinarsch tbd

File: docs/multi-agent-manager.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    "private_key_paths": {"fetchai": FET_PRIVATE_KEY_PATH_2},
+    "connection_private_key_paths": {"cosmos": COSMOS_PRIVATE_KEY_PATH_2}
+}
+manager.add_agent(weather_client_id, component_overrides=[component_overrides], agent_overrides=agent_overrides)
+```
+
+## Running the agents:
+
+``` python
+manager.start_agent(weather_station_id.name)
```
Comment (DavidMinarsch) [inline docs/multi-agent-manager.md:74]: Currently, I get this error:
```
>>> warning: [weather_station] ErrorHandler not initialized. Stopping AEA!
Stub connection receive error:
Traceback (most recent call last):
  File "fetchai/weather_station/vendor/fetchai/connections/stub/connection.py", line 253, in receive
    return await self.in_queue.get()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/asyncio/queues.py", line 159, in get
    await getter
concurrent.futures._base.CancelledError
```

File: docs/multi-agent-manager.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+## Adding projects
+
+We first add a couple of finished AEA project:
+
+``` python
+from aea.configurations.base import PublicId
+
+weather_client_id = PublicId.from_str("fetchai/weather_client:0.13.0")
+weather_station_id = PublicId.from_str("fetchai/weather_station:0.13.0")
```
Comment (DavidMinarsch) [inline docs/multi-agent-manager.md:25]: To run locally for testing, decrement version number by one

File: docs/multi-agent-manager.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+# c9b38eff57f678f5ab5304447997351edb08eceb883267fa4ad849074bec07e4
+COSMOS_PRIVATE_KEY_PATH_2 = "cosmos_private_key_2.txt"
+```
+
+Add the agent instances
+``` python
+agent_overrides = {
+    "private_key_paths": {"fetchai": FET_PRIVATE_KEY_PATH_1},
+    "connection_private_key_paths": {"cosmos": COSMOS_PRIVATE_KEY_PATH_1}
+}
```
Comment (DavidMinarsch) [inline docs/multi-agent-manager.md:49]: I don't think at present these get applied...

File: docs/multi-agent-manager.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+manager.add_agent(weather_station_id, agent_overrides=agent_overrides)
+
+component_overrides = {
+    "name": "p2p_libp2p",
+    "author": "fetchai",
+    "version": "0.9.0",
+    "type": "connection",
+    "config": {
+        "delegate_uri": "127.0.0.1:11001",
+        "entry_peers": ['/dns4/127.0.0.1/tcp/9000/p2p/16Uiu2HAkzgZYyk25XjAhmgXcdMbahrHYi18uuAzHuxPn1KkdmLRw'],
```
Comment (DavidMinarsch) [inline docs/multi-agent-manager.md:59]: only works if keys from above are used

File: docs/multi-agent-manager.md (hunk 5 above)
Comment (marcofavorito) [reply docs/multi-agent-manager.md:49]: indeed. Will fix now

**Review by lrahmani — COMMENTED** [review]
Suggestions

File: docs/multi-agent-manager.md (hunk 3 above)
Comment (DavidMinarsch) [reply docs/multi-agent-manager.md:74]: Version incompatibility of soef ..

**Pull request merged**